lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -979,8 +983,34 @@ def restart_all_services(self):
             )
         elif self._init_system == InitMode.DOCKER:
             self.exec_command("cd /home/vagrant/magma/lte/gateway/docker && docker-compose restart")
-        print("Waiting for all services to restart. Sleeping for 60 seconds..")
-        self.wait_for_restart_to_finish(60)
+
+        self._wait_for_pipelined_to_initialize()
```

Comment:
The intention and action of this function is to restart all the services. If we are calling to wait for pipelined explicitly, we must write a comment mentioning its need here (for keeping the track in future).

Reply:
You are right, we added some documentation.